app/grandchallenge/components/models.py

```diff
@@ -1929,3 +1929,87 @@ def update_size_in_storage(self):
         else:
             self.size_in_storage = self.image.size
             self.size_in_registry = self.calculate_size_in_registry()
+
+
+class CIVForObjectMixin:
+    def create_civ(self, ci_slug, new_value, user=None):
+        ci = ComponentInterface.objects.get(slug=ci_slug)
+        current_civ = self.values.filter(interface=ci).first()
```

Comment:
Please don't use the pattern of `filter` and `first`. It should be a `get` and then handling the error appropriately (which might mean 5xx as something has gone seriously wrong if there are two values for the same interface).